File: src/main/java/technology/touchmars/feign/wechat/client/api/MiniProgramUnionApiClient.java
package technology.touchmars.feign.wechat.client.api;

import org.springframework.cloud.openfeign.FeignClient;
import technology.touchmars.outside.feign.wechat.config.WechatMiniProgramCommonConfiguration;

import static technology.touchmars.feign.wechat.client.api.MiniProgramConstant.URL_BASE;

@FeignClient(name="miniProgramUnionApiClient", url=URL_BASE,
	configuration={WechatMiniProgramCommonConfiguration.class}
)
public interface MiniProgramUnionApiClient extends MiniProgramUnionApi {

}

File: src/main/kotlin/technology/touchmars/feign/wechat/client/api/MiniProgramUnionApiClient.kt
package technology.touchmars.feign.wechat.client.api

import org.springframework.cloud.openfeign.FeignClient
import technology.touchmars.outside.feign.wechat.config.WechatMiniProgramCommonConfiguration

@FeignClient(name = "miniProgramUnionApiClient", url = MiniProgramConstant.URL_BASE, configuration = [WechatMiniProgramCommonConfiguration::class])
interface MiniProgramUnionApiClient : MiniProgramUnionApi
